### ISSUE TITLE
Possible to run without `npm`?

### ISSUE THREAD

**Issue by tdmalone** [watson/test-all-versions#2]
I'm using Yarn, and don't have npm on my system at the moment.

Running tav (including through `yarn run`) gives me:

`Error: Command failed: npm -v`

which appears to be caused by [this line](https://github.com/watson/test-all-versions/blob/c2513b06a71fa68931f883edce9b29e5b42afafa/index.js#L20).

Would it be possible to skip this check and/or add in support for additional package managers?

**Comment by tdmalone** [comment]
Actually looking further... `spawn-npm-install` isn't going to work very well either, so this request might need to go upstream. ;)

**Comment by watson** [comment]
npm comes automatically with node, so it's normally a fair assumption that users of Node.js also have npm installed. What's the reason that you don't have it installed?

**Comment by tdmalone** [comment]
Yeah that's a fair point.

I removed npm so I don't accidentally `npm install`, because it would break the deterministic Yarn install and in some cases I've had to rm -rf node_modules and run yarn again. Having said that, I could just install it again and see how I go.

I think `spawn-npm-install` would potentially be the bigger problem, so I'll look further into this and consider an issue there if needed. Possibly that package should spawn `yarn` if a `yarn.lock` is present.

Thanks for your reply!
